RotatedBinarySearch.py

```diff
@@ -0,0 +1,39 @@
+#Time Complexity : O(logn)
+#Space Complexity : O(1)
+#Did this code successfully run on Leetcode : yes
+#Any problem you faced while coding this :
+
+
+# Your code here along with comments explaining your approach
+
+class Solution:
+    def search(self, nums, target) -> int:
+        if nums is None or len(nums) == 0:
+            return -1
+
+        size = len(nums)
+
+        left = 0
+        right = len(nums) - 1
+        # we will find the pivot point, where the array is shifted
+        while (left < right):
+            mid = (left + right) // 2
```

Comment:
try using (left + (right-left)/2), it'll help you avoid an integer overflow error

This is correct as well.